pages/markets/[marketid].tsx

```diff
@@ -122,6 +122,30 @@ const Market: NextPage<{
   const [hasAuthReport, setHasAuthReport] = useState<boolean>();
   const marketImageUrl = useMarketImageUrl(indexedMarket.img);
 
+  const { data: marketSdkv2 } = useMarket(Number(marketid));
+  const { data: marketStage } = useMarketStage(marketSdkv2);
+
+  const [scalarPrices, setScalarPrices] =
+    useState<{ short: number; long: number; type: ScalarRangeType }>();
+
+  useEffect(() => {
```

Comment:
I removed this code on this pr: https://github.com/zeitgeistpm/ui/pull/572/files#diff-9b9e5f6623dc512e77505d54e024c05291748f2101539a207cda21eed9413c9e

Could you double check the conflict resolution?